server/src/main/scala/com/cloudera/livy/utils/LineBufferedStream.scala

```diff
@@ -36,17 +38,17 @@ class LineBufferedStream(inputStream: InputStream) extends Logging {
   private val thread = new Thread {
     override def run() = {
       val lines = Source.fromInputStream(inputStream).getLines()
-      for (line <- lines) {
+       for (line <- lines) {
         _lock.lock()
         try {
-          _lines = _lines :+ line
+          info(s"stdout: $line")
+          if(logSize > 0) _lines add line
```

Comment:
Do we need to add `if` check, I assume if logSize is zero then we cannot add `line` to this queue.